app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php

```diff
@@ -23,7 +23,7 @@
 /**
  * Notify Customer of order pickup availability.
  */
-class NotifyPickup extends Action implements HttpPostActionInterface
+class NotifyPickup extends Action implements HttpGetActionInterface
```

Comment:
It seems to be POST action, not GET.
Could you fix it or create new issue to investigate?

Reply:
No - it is not POST

Reply:
You do not receive data.
You just make an action.
Why it is not 'POST' ?

Reply:
There is no choice. See the ship button.